addons/core/functions/intercom/fnc_getIntercomChannel.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_fnc_getIntercomChannel
+
+  Author: DartRuffian
+    Gets the channel of a player's vehicle intercom.
+
+  Arguments:
+    0: Vehicle <OBJECT>
+    1: Player <OBJECT>
+
+  Return Value:
+    The intercom channel <NUMBER>
+
+  Example:
+    [vehicle player, player] = call TFAR_fnc_getIntercomChannel;
+
+  Public: Yes
+*/
+
+params [
+    ["_vehicle", objNull, [objNull]],
+    ["_player", objNull, [objNull]]
+];
+
+if (isNull _vehicle || !isPlayer _player) exitWith {};
```

Comment:
Should this return `-1` or `-2` on early exit?

Returning a non-nil value would make it so third party mods don't need to explicitly check the player's vehicle, and could just call the function.
```sqf
if ([objectParent _unit, _unit] call TFAR_fnc_getIntercomChannel) >= 0) then {
    // _unit is a player, and is on an intercom
} else {
    // _unit is either not a player, or is not in a vehicle
};
```